data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ClientConfiguration.java

```diff
@@ -30,8 +30,8 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * A generic resolver from service names to {@link URI}s, using Spring yaml configuration
- * Uses Eureka if available, otherwise uses the yaml value directly as a URI (useful for k8s)
+* A generic resolver from service names to {@link URI}s
+* Uses Eureka if available, otherwise uses the Spring yaml configuration value directly as a URI (useful for k8s)
```

Comment:
```suggestion
 * A generic resolver from service names to {@link URI}s
 * Uses Eureka if available, otherwise uses the Spring yaml configuration value directly as a URI (useful for k8s)
```